src/ManifestStrategy/ConfigureModuleManifestStrategy.php

```diff
@@ -61,6 +61,16 @@ public function apply(array $manifest, string $moduleName, InputOutputInterface
                 );
             }
 
+            if (!is_bool($value) && !$value) {
```

Comment:
Hi @bohdan-turchyk-spryker, do we have a separate bug ticket for this case?

Reply:
https://spryker.atlassian.net/browse/PPLUS-1303 contains links to problematic manifests